core/crypto/signature/signature.cpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "crypto/signature/signature.hpp"
+#include "common/outcome.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(fc::crypto::signature, SignatureError, e) {
+  using fc::crypto::signature::SignatureError;
+  switch (e) {
+    case (SignatureError::INVALID_SIGNATURE_LENGTH):
+      return "Invalid signature length";
```

Comment:
```suggestion
      return "SignatureError: Invalid signature length";
```
Here and below. Just to be consistent.